apps/visualization/utils.py

```diff
@@ -314,17 +326,14 @@ def format_indicator_value(iso3, qs_map):
         ).values('iso3').annotate(
             indicator_value=Max('indicator_value'),
             month=F('indicator_month'),
+            format=F('format'),
         ).order_by('subvariable')
         country_most_recent_qs_iso3_map = {}
         for item in country_most_recent_qs:
             if country_most_recent_qs_iso3_map.get(item['iso3']):
-                country_most_recent_qs_iso3_map[item['iso3']].append(
-                    {'month': item['month'], 'indicator_value': item['indicator_value']}
-                )
+                country_most_recent_qs_iso3_map[item['iso3']].append(format_table_data(item))
```

Comment:
With 

```suggestion
                country_most_recent_qs_iso3_map[item['iso3']].append(OverviewTableDataType(**item))
```

Reply:
@thenav56  we have extra attribute 'iso3' in qs, but we don't put iso3 in OverviewTableDataType